File: решения/java/src/lab17/Employee.java
package lab17;

/**
 * Created by Александр on 20.07.2017.
 */
public class Employee {
    private String id;
    private String lastName;
    private String firstName;
    private String fatherName;
    private String gender;
    private String birthDate;
    private boolean marriage;
    private String postalIndex;
    private String address;
    private String position;
    private String hireType;
    private String notes;

    public String getId() {
        return id;
    }

    public String getLastName() {
        return lastName;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getFatherName() {
        return fatherName;
    }

    public String getGender() {
        return gender;
    }

    public String getBirthDate() {
        return birthDate;
    }

    public boolean isMarriage() {
        return marriage;
    }

    public String getPostalIndex() {
        return postalIndex;
    }

    public String getAddress() {
        return address;
    }

    public String getPosition() {
        return position;
    }

    public String getHireType() {
        return hireType;
    }

    public String getNotes() {
        return notes;
    }

    public Employee(String id, String lastName, String firstName, String fatherName, String gender, String birthDate, boolean marriage, String postalIndex, String address, String position, String hireType, String notes) {

        this.id = id;
        this.lastName = lastName;
        this.firstName = firstName;
        this.fatherName = fatherName;
        this.gender = gender;
        this.birthDate = birthDate;
        this.marriage = marriage;
        this.postalIndex = postalIndex;
        this.address = address;
        this.position = position;
        this.hireType = hireType;
        this.notes = notes;
    }
}

File: решения/cs/Classes/lab17/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Classes.lab17
{
    public class Employee
    {
        public String Id { private set; get; }
        public String LastName { private set; get; }
        public String FirstName { private set; get; }
        public String FatherName { private set; get; }
        public String Gender { private set; get; }
        public String BirthDate { private set; get; }
        public bool Marriage { private set; get; }
        public String PostalIndex { private set; get; }
        public String Address { private set; get; }
        public String Position { private set; get; }
        public String HireType { private set; get; }
        public String Notes { private set; get; }

        public Employee(String id, String lastName, String firstName, String fatherName,
            String gender, String birthDate, bool marriage, String postalIndex, String address,
            String position, String hireType, String notes)
        {
            Id = id;
            LastName = lastName;
            FirstName = firstName;
            FatherName = fatherName;
            Gender = gender;
            BirthDate = birthDate;
            Marriage = marriage;
            PostalIndex = postalIndex;
            Address = address;
            Position = position;
            HireType = hireType;
            Notes = notes;
        }
    }
}
